doc/api/console.markdown

```diff
@@ -48,6 +48,10 @@ is used on each argument.  See [util.format()][] for more information.
 
 Same as `console.log`.
 
+### console.debug([data][, ...])
```

Comment:
I suggest using the same signature as in the spec: `console.debug(object [,object, ...])`